tests/unittests/config/test_apt_configure_sources_list_v1.py

```diff
@@ -14,15 +14,6 @@
 EXAMPLE_TMPL = """\
 ## template:jinja
 ## Note, this file is written by cloud-init on first boot of an instance
-## modifications made here will not survive a re-bundle.
```

Comment:
Just reducing vertical space dedicated to unnecessary extra boilerplate test strings